modules/nw-openstack-external-ccm.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-cloud-config-reference.adoc
+// TODO: GitHub link clearance.
+
+:_content-type: CONCEPT
+[id="nw-openstack-external-ccm_{context}"]
+= The {rh-openstack} Cloud Controller Manager
```

Comment:
Should we remove RH reference from the CCM name as it's not provided/developed by Red Hat exclusively?
```suggestion
= The OpenStack Cloud Controller Manager
```

Reply:
@eurijon Probably yes, but I'm checking with our CS and DPM about this. I haven't dealt with this aspect of our branding practices before.